kanban-board/src/main/java/io/spine/examples/kanban/server/card/CardAggregate.java

```diff
@@ -43,96 +43,96 @@
 import java.util.Optional;
 
 /**
- * An aggregate representing a task.
+ * An aggregate representing a card.
  *
  * <p>Emits an {@link Pair event pair} in response to a command and an
  * {@link EitherOf4 either of four events} as a reaction to an outside event.
  *
- * <p>A task can be created with or without an assignee using {@link CreateTask create task 
- * command}. In response to it the aggregate emits a pair of events: {@link TaskCreated} and 
- * optionally {@link TaskAssigned} if the assignee is present.
+ * <p>A card can be created with or without an assignee using {@link CreateCard create card
+ * command}. In response to it the aggregate emits a pair of events: {@link io.spine.examples.kanban.card.event.CardCreated} and
+ * optionally {@link io.spine.examples.kanban.card.event.CardAssigned} if the assignee is present.
  *
- * <p>When a {@link TaskMoved task is moved} on the Kanban board its status changes emitting 
+ * <p>When a {@link io.spine.examples.kanban.card.event.CardMoved card is moved} on the Kanban board its status changes emitting
```

Comment:
```suggestion
 * <p>When a {@linkplain io.spine.examples.kanban.card.event.CardMoved card is moved} on the Kanban board its status changes emitting
```
And also due to change to absolute class path line got too long.